get/getAllItems/index.js

```diff
@@ -0,0 +1,24 @@
+var AWS = require('aws-sdk');
+var Joi = require('joi');
+
+AWS.config.update({region: 'us-east-2'});
+
+exports.handler = async (event) => {    
+    var ddb = new AWS.DynamoDB();    
+    var params = {
+        "TableName" : "Users"
+    };
+    console.log("Fetching all users from db...");
+    var result;
+    
+    try{
+        result = await ddb.scan(params).promise();
+        result = result.Items
+        
```

Comment:
Is it possible to be changed to:
```suggestion
(await ddb.scan(params).promise()).Items
```

Reply:
I agree, i will change it